identity-server/src/main/resources/com/hevelian/identity/server/api/errors/messages.properties

```diff
@@ -49,6 +49,10 @@ com.hevelian.identity.users.UserService$TenantAdminNotDeletableException.type =
 com.hevelian.identity.users.UserService$TenantAdminNotDeletableException.title = Tenant administrator '#{ex.userName}' cannot be deleted.
 com.hevelian.identity.users.UserService$TenantAdminNotDeletableException.detail = User '#{ex.userName}' is a tenant administrator of tenant '#{ex.tenant}' and cannot be deleted. Tenant administrator can be deleted only as part of the corresponding tenant.
 
+com.hevelian.identity.core.api.validation.TenantLogoValidator$IllegalLogoException.type = http://identity.hevelian.com/errors/tenant-logo-illegal-logo
```

Comment:
If you implement validation through a native mechanims (@Valid/@Validated) you will not need this exception at all. you will set a message and property directly in you validator. See examples.